docs/qase.md

```diff
@@ -0,0 +1,21 @@
+## [Qase-Reporter](../pkg/qase/README.md)
+## [Qase-Patch-Validation-Create](#qase-patch-validation-create)
+
+
+# Qase-Patch-Validation-Create
+
+### Description
+This job aims to automatically create a patch validation in Qase for a given project and test plan ID.
+
+### All needed variables are already defined or set in run time.
+
+- `QASE_API_TOKEN`: Qase API token to authenticate with the Qase API.
+- `QASE_PROJECT_CODE`: Qase project code where to create the patch validation.
+- `QASE_TEST_PLAN_ID`: Qase test plan ID used to create the patch validation.
+- `QASE_TAG`: Tag to filter the team the patch validation.
```

Comment:
This is a bit confusing to me, how and where are we using the `QASE_TAG`?

Reply:
Should mention that its hardcoded to `team-rke2`

Reply:
here is a overall doc, so doesnt need technical explanations